app/experimenter/jetstream/tests/test_tasks.py

```diff
@@ -39,7 +39,7 @@ def get_metric_data(self, data_point):
         ).dict(exclude_none=True)
 
     def add_outcome_data(self, data, overall_data, weekly_data, primary_outcome):
-        primary_metrics = ["mozilla_default_browser", "default_browser_action"]
+        primary_metrics = ["default_browser_action"]
```

Comment:
So the case we're looking for is where a metric is listed in the outcome but DOESN'T appear in the data right?  So we want to test the cases where it's 
- defined in the outcome and present in the data
- defined in the outcome and missing in the data
- missing in the outcome and present in the data

Which cases are we testing? Do we need to add/modify any other tests?

Reply:
Since we set the primary outcome in this test like so `primary_outcome = "default-browser"`, there are 2 metrics associated with it, `mozilla_default_browser` and `default_browser_action` based on the toml file. In this function, `add_outcome_data()` we only add one of the 2 outcome metrics to the mock jetstream data, this mimicks having Jetstream return data for `default_browser_action` but not for `mozilla_default_browser`. In the `constants.py` file I also removed the expectation that there will be an API response with data for `mozilla_default_browser`

So ..
`default_browser_action` is defined in the outcome and present in the data
`mozilla_default_browser` is defined in the outcome and missing in the data

Any other metric that is missing in the outcome and present in the data is treated as a default metric